src/main/java/mods/railcraft/common/carts/EntityCartChest.java

```diff
@@ -47,11 +50,6 @@ public boolean doInteract(EntityPlayer player, ItemStack stack, EnumHand hand) {
     }
 
     @Override
-    public EntityMinecart.Type getType() {
-        return EntityMinecart.Type.CHEST;
-    }
```

Comment:
This specific cart is intended to be for all intents and purposes seen as identical to the vanilla version of the chest cart. This won't change that will it?